go.mod

```diff
@@ -1,20 +1,35 @@
 module terraform-provider-confluent-schema-registry
 
-go 1.18
```

Comment:
Im not sure why the go version was updated here, I probably forgot to have my local version match what this package originally had